### PR TITLE
 print type and value in "flake attr is not a derivation" errors

### PR DESCRIPTION
**Pull request by Mic92**
# Motivation

Make `nix build` errors easier to understand

# Context

Source: https://git.lix.systems/lix-project/lix/commit/20981461d4a2a62c68f4bc7c4258473f7cd7d8e1  

# Priorities and Process

Add :+1: to [pull requests you find important](https://github.com/NixOS/nix/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc).

The Nix maintainer team uses a [GitHub project board](https://github.com/orgs/NixOS/projects/19) to [schedule and track reviews](https://github.com/NixOS/nix/tree/master/maintainers#project-board-protocol).

### CODE REVIEW

**Comment by lf-** [comment]
You could backport the release notes generator from Lix so the original change authors are credited in changelogs :P

File: doc/manual/rl-next/print-value-in-installable-flake-error.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+synopsis: New-cli flake commands that expect derivations now print the failing value and its type
```
Comment (fricklerhandwerk) [inline doc/manual/rl-next/print-value-in-installable-flake-error.md:2]:
```suggestion
synopsis: `nix flake` commands that expect derivations now print the failing value and its type
```

**Comment by Mic92** [comment]
> You could backport the release notes generator from Lix so the original change authors are credited in changelogs :P

Good idea. Do you want to make a pull request? I am still busy with other things.

File: doc/manual/rl-next/print-value-in-installable-flake-error.md (hunk 1 above)
Comment (Mic92) [reply doc/manual/rl-next/print-value-in-installable-flake-error.md:2]: Looks like the yaml parser doesn't like backticks. I had to remove them again.

**Review by roberth — APPROVED** [review]

**Pull request merged**